.travis.yml

```diff
@@ -39,7 +39,8 @@ services:
 # NOT on the travis vm space.
 python:
   - 3.7
```

Comment:
I would remove 3.7 from the version list so we only run with 1 version of python, unless we really need multiple versions here. Most stages/jobs explicitly list the python version, so I don't know that having an extra version is doing much at this time.